06_for_Assignment01.js

```diff
@@ -0,0 +1,35 @@
+console.log(`1. WAP to print numbers from 5 to 15 by incrementing by 1`);
+for (let index = 5; index <=15; index++) {
+    console.log( index);
+        
+    }
+console.log(`2. WAP to print numbers from 50 to 40 by decrementing by 1`);
+for (let index = 50; index >=40 ; index--) {
+    console.log( index);
+        
+    }
+console.log(`3. WAP to find first 15 odd numbers`);
+for (let index = 1; index <=15; index=index+2) {
```

Comment:
Have you verified this code that this will print 15 odd numbers?

Reply:
yes i have verified this code that this will print upto 15 odd numbers